### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nguyenth1**

### CODE REVIEW

**Comment by ccurtsinger** [comment]
Thank you Thu and Maddy.

Maddy: could you send me your GitHub username?

File: README.md [hunk 1]
```diff
@@ -3,37 +3,37 @@ Fill in the text below with your own answers to each question. Add your answers
...
-> ...
+> penalized 10% for each day
 
 What happens if you miss a quiz?
-> ...
+> Missed quizzes will receive a zero
 
 When are you free during my regularly scheduled office hours?
-> ...
+> Maybe (Maddy) and no (Thu)
```
Comment (ccurtsinger) [inline README.md:39]: Would you be able to make it to office hours at 9am (right after class)?

File: README.md [hunk 2]
```diff
@@ -3,37 +3,37 @@ Fill in the text below with your own answers to each question. Add your answers
...
-> ...
+> Thu
 
 Why are you taking this course?
-> ...
+> intended major
 
 What are your concerns about this course?
-> ...
+> workload
```
Comment (ccurtsinger) [inline README.md:15]: I hope the workload will not be too high for this class; I do expect there to be a lot of programming so you can learn the material by actually using it, but I won't be giving you much other work for this course.

File: README.md (hunk 1 above)
Comment (nguyenth1) [inline README.md:39]: Yes!

**Pull request closed without merging**